<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SamplerWindow</class>
 <widget class="QMainWindow" name="SamplerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>533</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Damn Sampler</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <property name="margin">
     <number>0</number>
    </property>
    <item>
     <widget class="QScrollArea" name="samplesArea">
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>768</width>
         <height>443</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QWidget" name="samplesWidget" native="true">
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <property name="margin">
            <number>0</number>
           </property>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave_state"/>
    <addaction name="actionRestore_state"/>
    <addaction name="actionClear_state"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_sample"/>
    <addaction name="actionEdit_sample"/>
    <addaction name="actionRemove_sample"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <widget class="QToolBar" name="samplesBar">
   <property name="windowTitle">
    <string>Samples</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>true</bool>
   </attribute>
   <addaction name="actionAdd_sample"/>
   <addaction name="actionEdit_sample"/>
   <addaction name="actionRemove_sample"/>
  </widget>
  <widget class="QToolBar" name="statesBar">
   <property name="windowTitle">
    <string>States</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave_state"/>
   <addaction name="actionRestore_state"/>
   <addaction name="actionClear_state"/>
  </widget>
  <widget class="QToolBar" name="controlBar">
   <property name="windowTitle">
    <string>Start-Stop</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionStart"/>
   <addaction name="actionStop"/>
  </widget>
  <action name="actionAdd_sample">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/add.png</normaloff>:/add.png</iconset>
   </property>
   <property name="text">
    <string>Add sample</string>
   </property>
  </action>
  <action name="actionRemove_sample">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/remove.png</normaloff>:/remove.png</iconset>
   </property>
   <property name="text">
    <string>Remove sample</string>
   </property>
  </action>
  <action name="actionEdit_sample">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/edit.png</normaloff>:/edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit sample</string>
   </property>
  </action>
  <action name="actionSave_state">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/save.png</normaloff>:/save.png</iconset>
   </property>
   <property name="text">
    <string>Save state</string>
   </property>
  </action>
  <action name="actionRestore_state">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/open.png</normaloff>:/open.png</iconset>
   </property>
   <property name="text">
    <string>Restore state</string>
   </property>
  </action>
  <action name="actionClear_state">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/clear.png</normaloff>:/clear.png</iconset>
   </property>
   <property name="text">
    <string>Clear state</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/settings.png</normaloff>:/settings.png</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/play.png</normaloff>:/play.png</iconset>
   </property>
   <property name="text">
    <string>Start!</string>
   </property>
   <property name="toolTip">
    <string>Start! [F5]</string>
   </property>
   <property name="shortcut">
    <string>F5</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop!</string>
   </property>
   <property name="toolTip">
    <string>Stop! [ESC]</string>
   </property>
   <property name="shortcut">
    <string>Esc</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
